external/Kconfig

```diff
@@ -9,3 +9,4 @@ source "$EXTERNALDIR/iotivity/Kconfig"
 source "$EXTERNALDIR/json/Kconfig"
 source "$EXTERNALDIR/libtuv/Kconfig"
 source "$EXTERNALDIR/wakaama/Kconfig"
+source "$EXTERNALDIR/RT-OCF/Kconfig"
```

Comment:
No. This is definitely wrong. It causes error on menuconfig. Please test it first and then try to merge.